css/pages/wishlist.css

```diff
@@ -0,0 +1,7 @@
+.dismiss-v{
+  position: absolute;
```

Comment:
suggestion - add more descriptive class name as dismiss-v doesn't give a context

Reply:
sure will make the changes. Thank you for appreciating the UI.